src/core/tsi/ssl_transport_security.cc

```diff
@@ -1062,8 +1061,8 @@ static tsi_result ssl_handshaker_create_frame_protector(
    * cannot call anything else but destroy on the handshaker after this call. */
```

Comment:
Please also update comment that transfer ownership of ssl and network_io to frame protector.